pkg/kcp/provider/gcp/nuke/new.go

```diff
@@ -0,0 +1,43 @@
+package nuke
+
+import (
+	"context"
+	"fmt"
+	"github.com/kyma-project/cloud-manager/api/cloud-control/v1beta1"
+	"github.com/kyma-project/cloud-manager/pkg/composed"
+	nuketypes "github.com/kyma-project/cloud-manager/pkg/kcp/nuke/types"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func New(stateFactory StateFactory) composed.Action {
+	return func(ctx context.Context, st composed.State) (error, context.Context) {
+
+		state, err := stateFactory.NewState(ctx, st.(nuketypes.State))
+		if err != nil {
+			obj := st.Obj().(*v1beta1.Nuke)
+			return composed.UpdateStatus(obj).
```

Comment:
Shouldn't this be patch since on all other places it is?
```suggestion
			return composed.PatchStatus(obj).
```